cognite_toolkit/_cdf_tk/load/_resource_loaders.py

```diff
@@ -638,50 +637,26 @@ def retrieve(self, ids: SequenceNotStr[str]) -> FunctionSchedulesList:
-        (_, bearer) = self.client.config.credentials.authorization_header()
-        created = FunctionSchedulesList([])
+        created = []
         for item in items:
-            if (
-                authentication := self.extra_configs.get(f"{item.function_external_id}:{item.cron_expression}", {}).get(
-                    "authentication"
-                )
-            ) is not None and len(authentication) > 0:
-                new_tool_config = CDFToolConfig()
-                old_credentials = cast(OAuthClientCredentials, new_tool_config.client.config.credentials)
-                new_tool_config.client.config.credentials = OAuthClientCredentials(
-                    client_id=authentication.get("clientId"),
-                    client_secret=authentication.get("clientSecret"),
-                    scopes=old_credentials.scopes,
-                    token_url=old_credentials.token_url,
-                )
-                session = get_oneshot_session(new_tool_config.client)
+            key = f"{item.function_external_id}:{item.cron_expression}"
+            auth_config = self.extra_configs.get(key, {}).get("authentication", {})
+            if "clientId" in auth_config and "clientSecret" in auth_config:
+                client_credentials = ClientCredentials(auth_config["clientId"], auth_config["clientSecret"])
             else:
-                session = get_oneshot_session(self.client)
-            nonce = session.nonce if session is not None else ""
-            try:
-                ret = self.client.post(
-                    url=f"/api/v1/projects/{self.client.config.project}/functions/schedules",
-                    json={
-                        "items": [
-                            {
-                                "name": item.name,
-                                "description": item.description,
-                                "cronExpression": item.cron_expression,
-                                "functionId": item.function_id,
-                                "data": item.data,
-                                "nonce": nonce,
-                            }
-                        ],
-                    },
-                    headers={"Authorization": bearer},
+                client_credentials = None
+
+            created.append(
+                self.client.functions.schedules.create(
+                    name=item.name or "",
+                    description=item.description or "",
+                    cron_expression=cast(str, item.cron_expression),
+                    function_id=cast(int, item.function_id),
+                    data=item.data,
+                    client_credentials=client_credentials,
                 )
-            except CogniteAPIError as e:
-                if e.code == 400 and "Failed to bind session" in e.message:
-                    print("  [bold yellow]WARNING:[/] Failed to bind session because function is not ready.")
-                continue
```

Comment:
Reason for removing: The session binds just fine for a function that is still deploying. I guess the bind fails if an invalid nonce is passed which was possible before, see line (also removed):
`nonce = session.nonce if session is not None else ""`